scripts/juju-inspect/rules/startcount.go

```diff
@@ -0,0 +1,69 @@
+package rules
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+)
+
+type StartCountRule struct {
+	counts map[string]map[string]int
+}
+
+func NewStartCountRule() *StartCountRule {
+	return &StartCountRule{
+		counts: make(map[string]map[string]int),
+	}
+}
+
+func (r *StartCountRule) Run(name string, report Report) {
+	for manifoldName, manifold := range report.Manifolds {
+		if _, ok := r.counts[name]; !ok {
+			r.counts[name] = make(map[string]int)
+		}
+		r.counts[name][manifoldName] = manifold.StartCount
+	}
+}
+
+func (r *StartCountRule) Summary() string {
+	return "Start Counts:"
+}
+
+type namedCount struct {
+	name  string
+	count int
+}
+
+func (r *StartCountRule) Analyse() string {
+	// Gather
+	total := make(map[string]int, len(r.counts))
+	highest := make(map[string]namedCount, len(r.counts))
+	for ctrl, manifolds := range r.counts {
+		for name, v := range manifolds {
+			total[ctrl] += v
+
+			nc := highest[ctrl]
+			if v > nc.count {
```

Comment:
I wonder if it would be better to have a list of highest, rather than just highest?